File: Soe.Common/DTO/EmployeeVacationPrelUsedDaysDTO.cs
namespace SoftOne.Soe.Common.DTO
{
    public class EmployeeVacationPrelUsedDaysDTO
    {
        public int EmployeeId { get; set; }
        public decimal Sum { get; set; }
        public bool IsHours { get; set; }
        public string Details { get; set; }
    }
}

File: Soe.Angular/TypeScript/Common/Models/EmployeeVacationPrelUsedDaysDTO.ts
import { IEmployeeVacationPrelUsedDaysDTO } from "../../Scripts/TypeLite.Net4";

export class EmployeeVacationPrelUsedDaysDTO implements IEmployeeVacationPrelUsedDaysDTO {
	details: string;
	employeeId: number;
	isHours: boolean;
	sum: number;
}
